package.json

```diff
@@ -12,27 +12,25 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && cross-env nuxt --hostname 0.0.0.0",
+    "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && nuxt --hostname 0.0.0.0",
     "build": "pnpm i18n:get-translations && nuxt build",
     "build-only": "nuxt build",
     "generate": "nuxt generate",
     "start": "nuxt start",
     "prod": "pnpm build-only && pnpm start",
     "storybook": "nuxt storybook",
     "test": "jest",
-    "test:all": "pnpm test:e2e & jest",
+    "test:all": "pnpm test:playwright & jest",
     "test:watch": "jest --collectCoverage=false --watch",
     "talkback": "node ./test/proxy.js",
-    "pretest:e2e": "pnpm install",
-    "test:e2e": "cross-env PLAYWRIGHT_VERSION=1.17.1 TEST_TYPE=e2e docker-compose -f docker-compose.playwright.yml up --abort-on-container-exit --exit-code-from playwright",
-    "test:e2e:local": "playwright test -c test/e2e",
-    "test:e2e:debug": "cross-env PWDEBUG=1 pnpm test:e2e:local",
-    "test:e2e:recreate-tapes": "npm-run-all delete-tapes test:e2e:update-tapes --continue-on-error",
-    "test:e2e:update-tapes": "npm-run-all update-tapes fix-tape-permissions --continue-on-error",
+    "playwright-version": "pnpm ls --dev --depth=0 | grep playwright | awk '{print $2}'",
+    "pretest:playwright": "pnpm install",
+    "test:playwright": "USER_ID=${USER_ID:-$(id -u)}; PLAYWRIGHT_VERSION=$(pnpm playwright-version --silent) docker-compose -f docker-compose.playwright.yml up --abort-on-container-exit --exit-code-from playwright",
```

Comment:
The semicolon is necessary for being able to programatically set the `USER_ID` variable including with a default. the `${}` syntax with a default value doesn't work if you do the `VAR=[value] [command]` syntax directly.